custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssuePage.kt

```diff
@@ -60,6 +59,16 @@ class CloudIssuePage(
         return CloudIssueLinking(driver)
     }
 
+    override fun addAttachment() {
+        val src = (driver as TakesScreenshot).getScreenshotAs(OutputType.BYTES)
+       val builder = Actions(driver)
+        builder.keyDown(Keys.CONTROL).sendKeys("v").perform()
+        driver.wait(ExpectedConditions.visibilityOfElementLocated(By.xpath("(//*[@data-test-id=" +
+            "'issue.views.issue-base.content.attachment.filmstrip-panel']" +
+            "//span[contains(text(),'Attachments')])")))
```

Comment:
We don't have to care if it's `span`
```suggestion
            "//*[contains(text(),'Attachments')])")))
```

Reply:
changed